### DISCUSSION TITLE
Not sure if this is a bug... re: algorithm recycling "easy" too soon

### DISCUSSION THREAD

**Discussion by MatthewJMullins** [st3v3nmw/obsidian-spaced-repetition#182] [Q&A]
It's tough for me to reproduce. What I was noticing upon making a whole bunch of new cards (112 of them) -- even though I was marking some cards as "easy 3.5 days", they were coming up again. Sometimes "easy" cards were cycling back even 4 cards down the line. I'm not sure if this is expected behavior and/or I don't understand the genius behind the Anki-like algorithm. Easy cards cycling back isn't intuitive to me. Anyone else noticing this?

**Comment by st3v3nmw** [comment]
I think it's because we're load balancing too early. I'll set a higher threshold.
